cmd/antrea-agent/options.go

```diff
@@ -320,9 +320,12 @@ func (o *Options) validateFlowExporterConfig() error {
 	return nil
 }
 
-func (o *Options) validateMulticastConfig() error {
+func (o *Options) validateMulticastConfig(encryptionMode config.TrafficEncryptionModeType) error {
 	if features.DefaultFeatureGate.Enabled(features.Multicast) {
```

Comment:
looking at your latest code, we should actually add `o.config.Multicast.Enable` to the condition. Otherwise, this change is not user friendly at all: when installing Antrea (either using the manifest or the Helm chart), users will need to _explicitly_ disable the Multicast FeatureGate. That should not be required. Instead, we should only return validation errors if WireGuard is enabled AND Multicast was explicitly enabled using the boolean toggle.

Reply:
Got it, will update.